app/views/manage_assessments/assignments/new.html.erb

```diff
@@ -1,5 +1,45 @@
-<%= simple_form_for @assignment, url: manage_assessments_outcome_coverage_assignments_path do |form| %>
-  <%= form.input :name, label: t(".assignment") %>
-  <%= form.input :problem, label: t(".problem") %>
+<%= content_for :full_bleed do %>
+  <div class="madlib-header">
+    <h1 class="headline-narrow">
+      Add assignment
+    </h1>
+    <h2 class="headline-narrow-subhead">
+      <%= @assignment.outcome_coverage.coverage.course.number %>
+      &middot;
+      <%= @assignment.outcome_coverage.coverage.course.name %>
+    </h2>
+  </div>
+<% end %>
+
+
+<%= simple_form_for @assignment, html: { class: "madlib" }, url: manage_assessments_outcome_coverage_assignments_path do |form| %>
+  <%= form.input :name,
+      label: t(".assignment"),
+      placeholder: "Problem Set 4" %>
+  <%= form.input :problem,
+      label: t(".problem"),
+      placeholder: "Problem 4"%>
+
   <%= form.submit t(".add") %>
+  <%= link_to t("helpers.cancel_button"), :back %>
 <% end %>
+
+<script type = "text/javascript">
+  $(function() {
+    var inp = $("input[type='text']");
+
+    function err() {
+      $.each(inp, function() {
+        if (!$.trim(this.value)) {
+          $(this).addClass("is-empty");
+          $(this).removeClass("has-items");
+        } else {
+          $(this).removeClass("is-empty");
+          $(this).addClass("has-items");
+        }
+      });
+    }
+
+    inp.blur(err);
```

Comment:
Do you want to do this only on blur, or would you like it done any time the user enters text (don't wait until they blur)?